recipes/r-pastclim/meta.yaml

```
@@ -0,0 +1,91 @@
+{% set version = '2.1.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-pastclim
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/pastclim_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/pastclim/pastclim_{{ version }}.tar.gz
+  sha256: 53e15d3dc0f9b61be7ab8ca2d6dc9379b414cb3f597b919d601c01b4ec62ac71
+
+build:
+  skip: true  # [win]
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-curl
+    - r-lubridate
+    - r-ncdf4
+    - r-sf
+    - r-terra >=1.7.18
+    - r-xml2
+  run:
+    - r-base
+    - r-curl
+    - r-lubridate
+    - r-ncdf4
+    - r-sf
+    - r-terra >=1.7.18
+    - r-xml2
+
+test:
+  commands:
+    - $R -e "library('pastclim')"           # [not win]
+    - "\"%R%\" -e \"library('pastclim')\""  # [win]
+
+about:
+  home: https://github.com/EvolEcolGroup/pastclim, https://evolecolgroup.github.io/pastclim/
```

Let's split these URLs:

```suggestion
  home: https://evolecolgroup.github.io/pastclim/
  dev_url: https://github.com/EvolEcolGroup/pastclim
```